tables/rx_antibodies/ikemura21-mab.csv

```diff
@@ -1,3 +1,4 @@
 ref_name,rx_name,ab_name
-Ikemura21,REGN10987+REGN10933,Imdevimab+Casirivimab
+Ikemura21,REGN10987+REGN10933,Imdevimab
+Ikemura21,REGN10987+REGN10933,Casirivimab
 Ikemura21,GSK4182136,Sotrovimab
```

Comment:
the rx_name Sotrovimab is used in rx_potency.csv, so you can change GSK4182136 to Sotrovimab in ikemura21-mab.csv.